stdlib/tests/crypto/stark/mod.rs

```diff
@@ -386,101 +389,117 @@ const TEST_RANDOM_INDICES_GENERATION: &str = r#"
 
 /// This is an output of the ACE codegen in AirScript and encodes the circuit for executing
 /// the constraint evaluation check i.e., DEEP-ALI.
```

Comment:
Not sure how long we'll keep this, but would be good to add instructions on how to generate this (e.g. I wouldn't know how)